django_filters/fields.py

```diff
@@ -71,3 +73,25 @@ def compress(self, data_list):
         if len(data_list)==2:
             return Lookup(value=data_list[0], lookup_type=data_list[1] or 'exact')
         return Lookup(value=None, lookup_type='exact')
+
+
+class IsoDateTimeField(forms.DateTimeField):
+    """
+    Supports 'iso-8601' date format too which is out the scope of
+    the ``datetime.strptime`` standard library
+
+    # ISO 8601: ``http://www.w3.org/TR/NOTE-datetime``
+
+    Based on Gist example by David Medina https://gist.github.com/copitux/5773821
+    """
+    ISO_8601 = 'iso-8601'
+    input_formats = [ISO_8601]
+
+    def strptime(self, value, format):
```

Comment:
This is just the method signature from the (Django) superclass. (But, yes, I see your point.)